### ISSUE TITLE
Broken links in `EUC_JP` encoding doc

### ISSUE THREAD

**Issue by Mingun** [hsivonen/encoding_rs#86]
[EUC_JP doc](https://docs.rs/encoding_rs/latest/encoding_rs/static.EUC_JP.html) refers to the [euc-jp.html](https://encoding.spec.whatwg.org/euc-jp.html) and [euc-jp-bmp.html](https://encoding.spec.whatwg.org/euc-jp-bmp.html), which are not exist. According to the https://encoding.spec.whatwg.org/#indexes I assume, that the correct link is https://encoding.spec.whatwg.org/jis0212.html and https://encoding.spec.whatwg.org/jis0212-bmp.html

**Comment by hsivonen** [comment]
Yeah, EUC-JP should link to the pages you point out and also to https://encoding.spec.whatwg.org/jis0208.html and https://encoding.spec.whatwg.org/jis0208-bmp.html . Looks like I need to revive a way to run the Python 2-dependent generation script.

**Comment by hsivonen** [comment]
Thank you. Now fixed.
